redis-om-spring/src/main/java/com/redis/om/spring/RedisModulesConfiguration.java

```diff
@@ -229,17 +238,23 @@ public HuggingFaceTokenizer sentenceTokenizer(RedisOMSpringProperties properties
         "maxLength", properties.getDjl().getSentenceTokenizerMaxLength(), //
         "modelMaxLength", properties.getDjl().getSentenceTokenizerModelMaxLength() //
     );
-    return HuggingFaceTokenizer.newInstance(properties.getDjl().getSentenceTokenizerModel(), options);
+    try {
+      InetAddress.getByName("www.google.com").isReachable(5000);
```

Comment:
Should we check the connection for the exact urls we need to reach ? as google.com can be reachable in such cases ...